recipes/sagemaker-code-editor/build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+# This code includes code from https://github.com/conda-forge/openvscode-server-feedstock
+# which is licensed under the BSD-3-Clause License.
+
+export NODE_OPTIONS=--openssl-legacy-provider
+
+# ripgrep tries to download binary from github, copying the ripgrep executable to the cache location so that it doesn't do that
+mkdir /tmp/vscode-ripgrep-cache-1.15.5
+mkdir /tmp/vscode-ripgrep-cache-1.15.6
+cp "${PREFIX}/bin/rg" /tmp/vscode-ripgrep-cache-1.15.5/
+cp "${PREFIX}/bin/rg" /tmp/vscode-ripgrep-cache-1.15.6/
+
+pushd sagemaker-code-editor
+pushd src
+
+# Install build tools for build_platform
+(
+  export CFLAGS="-isystem ${BUILD_PREFIX}/include -O2"
+  export CXXFLAGS="-isystem ${BUILD_PREFIX}/include -O2"
+  export CPPFLAGS="-isystem ${BUILD_PREFIX}/include -O2"
```

Comment:
I think `-O2` is already added in the [default config](https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/9661e4c851131527fab6a895ceb567b5172b9d48/recipe/conda_build_config.yaml#L53).

Not sure why you need `-isystem` from BUILD_PREFIX when it should come from PREFIX 🤔

Reply:
Context - we referred to another similar conda recipe as a starting point - this particular block of code was added there for cross-compile support which we don't need - removed it